Dry Run on First Migration Fails

When I try using --dry-run during my first migration, I get this.

$ node_modules/.bin/db-migrate up -v --config config/database.json -e dev --dry-run
[INFO] dry run
[INFO] Using dev settings: { driver: 'pg', database: '***' }
[INFO] require: ./pg
[INFO] connecting
[INFO] connected
[SQL] select version() as version
select version() as version
[ERROR] TypeError: Cannot read property 'rows' of undefined

--- a/lib/driver/pg.js
+++ b/lib/driver/pg.js
@@ -57,7 +57,7 @@ var PgDriver = Base.extend({
           return callback(err);
         }
 
-        if (result.rows && result.rows.length > 0 && result.rows[0].version) {
+        if (result && result.rows && result.rows.length > 0 && result.rows[0].version) {
           var version = result.rows[0].version;
           var match = version.match(/\d+\.\d+\.\d+/);
           if (match && match[0] && semver.gte(match[0], '9.1.0')) {
@@ -70,7 +70,7 @@ var PgDriver = Base.extend({
             return callback(err);
           }
 
-          if (result.rows && result.rows.length < 1) {
+          if (result && result.rows && result.rows.length < 1) {
             this.createTable('migrations', options, callback);
           } else {
             callback();

